src/test/java/com/notably/commons/core/Path/AbsolutePathTest.java

```diff
@@ -0,0 +1,38 @@
+package com.notably.commons.core.path;
+
+import static com.notably.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+
+class AbsolutePathTest {
+
+    @Test
+    public void createAbsolutePath() {
+        assertThrows(InvalidPathException.class, () -> AbsolutePath.fromString("CS2103/notes"));
+        try {
+            assertEquals(new AbsolutePath("/CS2103/notes"), AbsolutePath.fromString("/CS2103/notes"));
+        } catch (InvalidPathException ex) {
+            System.out.println("Error");
+        }
```

Comment:
You should split this into two test cases:
1. One, you assert that without the leading `"/"` it's gonna throw
2. Another one, you assert that with the correct string the path got created successfully.